layers/place/style.json

```diff
@@ -81,6 +84,207 @@
+          [
+            "get",
+            "name:en"
+          ],
+          [
+            "get",
+            "name_en"
+          ],
+          [
+            "get",
+            "name:latin"
+          ]
+        ],
+        "text-font": [
+          "Noto Sans Bold"
+        ],
+        "text-letter-spacing": 0.1,
+        "text-max-width": 9,
+        "text-size": {
+          "base": 1.2,
+          "stops": [
+            [
+              5,
+              10
+            ],
+            [
+              15,
+              14
+            ]
+          ]
+        },
+        "text-transform": "uppercase",
+        "visibility": "visible",
+        "symbol-placement": "point",
+        "text-variable-anchor": [
+          "center",
+          "top-right",
+          "right",
+          "bottom-right",
+          "bottom",
+          "bottom-left",
+          "left",
+          "top-left",
+          "top"
+        ],
+        "text-radial-offset": 0.1,
+        "text-padding": {
+          "base": 5,
+          "stops": [
+            [
+              3,
+              15
+            ],
+            [
+              10,
+              15
+            ],
+            [
+              15,
+              25
+            ]
+          ]
+        }
+      },
+      "paint": {
+        "text-halo-color": "rgba(255,255,255,0.8)",
+        "text-halo-width": 1.2,
+        "text-color": "rgba(89,77,76,1)"
+      },
+      "order": 104
+    },
+    {
+      "id": "place-state-1",
+      "type": "symbol",
+      "metadata": {
+        "mapbox:group": "1444849242106.713"
+      },
+      "source": "openmaptiles",
+      "source-layer": "place",
+      "minzoom": 0,
+      "filter": [
+        "all",
+        [
+          "in",
+          "class",
+          "state"
+        ],
+        [
+          "<=",
+          "rank",
+          3
+        ]
+      ],
+      "layout": {
+        "text-field": [
+          "coalesce",
+          [
+            "get",
+            "name:en"
+          ],
+          [
+            "get",
+            "name_en"
+          ],
+          [
+            "get",
+            "name:latin"
+          ]
+        ],
+        "text-font": [
+          "Noto Sans Bold"
+        ],
+        "text-letter-spacing": 0.1,
+        "text-max-width": 9,
+        "text-size": {
+          "base": 1.2,
+          "stops": [
+            [
+              0,
+              7
+            ],
+            [
+              4,
+              10
+            ],
+            [
+              15,
+              14
+            ]
+          ]
+        },
+        "text-transform": "uppercase",
+        "visibility": "visible",
+        "symbol-placement": "point",
+        "text-variable-anchor": [
+          "center",
+          "top-right",
+          "right",
+          "bottom-right",
+          "bottom",
+          "bottom-left",
+          "left",
+          "top-left",
+          "top"
+        ],
+        "text-radial-offset": 0.1,
```

Comment:
IMO, it's better to center the text label on the label node for state/provinces which is the default if we remove these properties.

```suggestion
```